qc_grader/challenges/programming_challenge_aus/problem_set.py

```diff
@@ -1,62 +1,46 @@
-import networkx as nx
-
-from typeguard import typechecked
-
 from typing import Callable, Dict
 
+import networkx as nx
 from qiskit import QuantumCircuit
-
-from qc_grader.grader.grade import (
-    grade, get_problem_set
-)
-from qc_grader.grader.common import (
-    circuit_to_json
-)
-
 from qiskit.algorithms.minimum_eigensolvers.sampling_vqe import SamplingVQEResult
 from qiskit_optimization.applications import Tsp
+from typeguard import typechecked
+
+from qc_grader.grader.common import circuit_to_json
+from qc_grader.grader.grade import get_problem_set, grade
 
-_challenge_id = 'programming_challenge_aus'
+_challenge_id = "programming_challenge_aus"
 
 
 @typechecked
 def grade_prob1_ex3(qc: QuantumCircuit, counts: Dict[str, int]) -> None:
-    answer = {
-        'qc': circuit_to_json(qc),
-        'counts': counts
-    }
+    answer = {"qc": circuit_to_json(qc), "counts": counts}
 
-    grade(answer, 'prob1-ex3', _challenge_id)
+    grade(answer, "prob1-ex3", _challenge_id)
 
 
 @typechecked
 def grade_prob2_ex1(tsp_function: Callable) -> None:
-    _, inputs = get_problem_set('prob2-ex1', _challenge_id)
+    _, inputs = get_problem_set("prob2-ex1", _challenge_id)
 
     answer = {}
     for i in inputs.keys():
-        distance, order = tsp_function(inputs[i], int(i))
-        answer[i] = {
-            'distance': distance,
-            'order': order
-        }
+        distance, order = tsp_function(int(i), inputs[i])
```

Comment:
Main change here

Reply:
Ah I didn't see the template notebook. Yes I think we need to keep it as per the template. Maybe this PR is not needed at all then! :D